src/components/Events/Events.js

```diff
@@ -6,10 +6,18 @@ import { EventsList, EventFilters, Loading } from '../';
 import { eventsAPI } from '../../api';
 import styles from './Events.sass';
 
+const odataFilterTypes = ['searchText', 'startDate'];
+const nonOdataFiltertypes = ['location', 'range'];
+
 function hasMoreEventsToLoad(currentPage, totalPages) {
   return currentPage + 1 < totalPages;
 }
 
+function buildFilterValues(filters, filterTypes) {
```

Comment:
The object being built up here is specific to the current state of the filters so I don't think it would fit anywhere into the events API.  I'll include an example though